### PR TITLE
add libkml

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://github.com/libkml/libkml

### CODE REVIEW

File: recipes/libkml/build.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX \
```
Comment (jschueller) [inline recipes/libkml/build.sh]: please use CMAKE_PREFIX_PATH

File: recipes/libkml/build.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+      $SRC_DIR
+ctest
+make install
```
Comment (jschueller) [inline recipes/libkml/build.sh]: use CPU_COUNT

Comment (ocefpaf) [inline recipes/libkml/build.sh]: This is not really heavy to build but it does not hurt to use it. Thanks!

File: recipes/libkml/build.sh (hunk 1 above)
Comment (ocefpaf) [inline recipes/libkml/build.sh]: For some reason that did not work for me locally.

File: recipes/libkml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  skip: True  # [win]
+
+requirements:
+  build:
+    - cmake
+    # - gtest
+    # - uriparser
+    # - minizip
+    - expat
+    - zlib
```
Comment (isuruf) [inline recipes/libkml/meta.yaml]: Needs pinning.

File: recipes/libkml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - cmake
+    # - gtest
+    # - uriparser
+    # - minizip
+    - expat
```
Comment (isuruf) [inline recipes/libkml/meta.yaml]: Maybe this should be pinned. https://abi-laboratory.pro/tracker/timeline/expat/

File: recipes/libkml/meta.yaml (hunk 3 above)
Comment (ocefpaf) [inline recipes/libkml/meta.yaml]: :+1:

File: recipes/libkml/meta.yaml (hunk 4 above)
Comment (ocefpaf) [inline recipes/libkml/meta.yaml]: Probably. We have only 1 version, but we will have trouble if we update it.

**Comment by ocefpaf** [comment]
Thanks @jschueller and @isuruf for the review!

**Pull request merged**